CMakeLists.txt

```diff
@@ -21,6 +21,21 @@ CMAKE_CACHEFILE_DIR: ${CMAKE_CACHEFILE_DIR}
 ")
 endif()
 
+
+# See https://gitlab.kitware.com/cmake/cmake/issues/16228
+# and https://cmake.org/pipermail/cmake/2019-May/thread.html#69496
+#
+if(NOT ZEPHYR_BASE STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+message(FATAL_ERROR "ZEPHYR_BASE doesn't match CMAKE_CURRENT_SOURCE_DIR
+  ${ZEPHYR_BASE}\t= ZEPHYR_BASE
```

Comment:
Generally, this kind of variable/value display would list the variable name first then its value (e.g., printenv comment)

Reply:
That's what I did initially but it made the values much more difficult to compare.